Assets/Tests/Unit/VimeoPlayerTest.cs

```diff
@@ -0,0 +1,63 @@
+using UnityEngine;
+using UnityEngine.TestTools;
+using NUnit.Framework;
+using System;
+using System.Collections;
+using Vimeo.Player;
+
+public class VimeoPlayerTest : TestConfig {
```

Comment:
I am unable to find the class `TestConfig` - Unity 2018.2.6f1
<img width="1173" alt="screen shot 2018-09-26 at 10 24 50 am" src="https://user-images.githubusercontent.com/15909884/46087539-8b2cb780-c178-11e8-97a8-6210bbf0b166.png">

Also I think our style is new line before opening brackets on class definitions, right?
```csharp
public class VimeoPlayerTest : TestConfig 
{
}
```

Reply:
Ignore my `TestConfig` comment